### ISSUE TITLE
[Bug]: Bad text contrast in selection bar

### ISSUE THREAD

**Issue by kalwados** [nextcloud/desktop#8361]
- [x] This is a **bug**, not a question or a configuration issue.
- [x] This issue is **not** already reported on Github (I've searched it).
- [x] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

Windows 11: The selection bar in main window provides bad contrast for the text; see screenhot. 
...

### Steps to reproduce

- Windows Task bar: righ click the Nextcloud icon
- click "open main window"
- hoover the mouse cursor over the items
- on selected item, the text contrast is bad: black text on dark blue background.

...

### Expected behavior

The selection bar should use white text color for selection bar (or what windows defines as default colors for selection bar and selected text)
...

### Which files are affected by this bug

n/a

### Operating system

Windows

### Which version of the operating system you are running.

Windows 11

### Package

Official Windows MSI

### Nextcloud Server version

30.10.11

### Nextcloud Desktop Client version

3.16.5

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [x] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell

```

### Additional info

![Image](https://github.com/user-attachments/assets/e41447ac-8304-4794-9986-10a518690afa)

**Comment by barisbasar1209** [comment]
@nilsding @kalwaydos this issue appears to be an exact duplicate of issue #8221, which I am currently seeking to resolve. 
Should they be merged?

**Comment by nilsding** [comment]
@barisbasar1209 true that, will close this as a duplicate

thank you :)

**Comment by kalwados** [comment]
> [@nilsding](https://github.com/nilsding)this issue appears to be an exact duplicate of issue [#8221](https://github.com/nextcloud/desktop/issues/8221), which I am currently seeking to resolve. Should they be merged?

Yes; it's a duplicate.
